scalyr_agent/builtin_monitors/docker_monitor.py

```diff
@@ -596,14 +692,23 @@ def __get_last_request_for_log( self, path ):
 
         return scalyr_util.seconds_since_epoch( result )
 
-    def __create_log_config( self, parser, path, attributes, parse_as_json=False ):
+    def __create_log_config( self, parser, path, attributes, base_config={}, parse_as_json=False ):
         """Convenience function to create a log_config dict from the parameters"""
```

Comment:
Can you add in some documentation for the arguments.

Reply:
Done